__tests__/pet.test.js

```diff
@@ -0,0 +1,112 @@
+const Pet = require("../src/pet");
+
+describe("constructor", () => {
+  const pet = new Pet("Fido");
+  it("returns an object", () => {
```

Comment:
It would be good to separate each `it` block by a clear line to make it a bit easier to read